Foundation/NSFileManager.swift

```diff
@@ -590,10 +585,10 @@ public class NSFileManager : NSObject {
         return self.fileExistsAtPath(path, isDirectory: nil)
     }
     
-    public func fileExistsAtPath(path: String, isDirectory: UnsafeMutablePointer<ObjCBool>) -> Bool {
+    public func fileExistsAtPath(path: String, isDirectory: UnsafeMutablePointer<ObjCBool>?) -> Bool {
         var s = stat()
         if lstat(path, &s) >= 0 {
-            if isDirectory != nil {
+            if let isDirectory = isDirectory {
```

Comment:
I introduced a few of these `if let x = x`. This has been a controversial idiom in the Swift community, so please let me know if you'd prefer alternate names for the unwrapped value every time.